problems/strings.js

```diff
@@ -87,14 +123,40 @@ function removeLastWord() {}
  * @returns {string}
  */
 
-function everyOtherLetter() {}
+function everyOtherLetter(str) {
+  let output = "";
+  for (let i = 0; i < str.length; i++) {
+    if (i % 2 === 0) {
+      output += str[i];
+    }
+  }
+  return output;
+}
 
 /**
  * Takes in a string and returns a new string that has all the vowels removed.
  * @param {string}
  * @returns {string}
  */
-function disemvowel() {}
+function disemvowel(str) {
+  let vowels = {
+    'a' : true,
+    'e' : true, 
+    'i' : true,
+    'o' : true,
+    'u' : true, 
+  };
+
+  let output = ""
+  for (let i = 0; i < str.length; i++){
+    let letter = str[i].toLowerCase();
+    if (!vowels[letter]) {
```

Comment:
Could make vowels like
let vowels = "aeiou" 
and check !vowels.includes(letter) 
Might be cleaner